# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/poetry/hotfix-gpep.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/pyproject.toml.orig	2025-02-01 18:01:19.844120623 +0200
+++ bb/pyproject.toml	2025-02-01 18:00:38.800785262 +0200
@@ -57,6 +57,10 @@
 
 
 [tool.poetry]
+name = "poetry"
+version = "2.0.1"
+description = "A one more python package build system in universe"
+authors = ["Your Name <your.email@example.com>"]
 requires-poetry = ">=2.0"
 classifiers = [
     "Topic :: Software Development :: Build Tools",
